src/transformers/models/rwkv/modeling_rwkv.py

```diff
@@ -159,7 +159,7 @@ def forward(ctx, time_decay, time_first, key, value, state=None, return_state=Fa
 
     @staticmethod
     # g stands for grad
-    def backward(ctx, g_output):
+    def backward(ctx, g_output, g_state=None):
```

Comment:
why the variable is not used later on?

Reply:
Because we don't handle the gradient of the state. But autograd is not happy if this isn't here.

Reply:
Thanks so much for explaning!